tasks/node_rm.rb

```diff
@@ -6,9 +6,9 @@
 require 'puppet'
 
 def node_rm(force, node)
-  cmd_string = 'docker node rm'
-  cmd_string += ' --force' unless force.nil?
-  cmd_string += " #{node}" unless node.nil?
+  cmd_string = ['docker', 'node', 'rm']
+  cmd_string += [' --force'] unless force.nil?
+  cmd_string += [" #{node}"] unless node.nil?
```

Comment:
We should try to avoid using += here.

I believe [Array#concat](https://ruby-doc.org/core-2.1.1/Array.html#method-i-concat) would be preferable in this instance.

Also lets rename the variable to cmd rather than cmd_string.

Reply:
See above.

Reply:
In this case you have changed from concatenating a string to an array so we should generally lean towards the more idiomatic approach.